Makefile

```diff
@@ -25,6 +25,12 @@ proto:
 	for dir in $$(git ls-files '*.proto' | xargs -n1 dirname | uniq); do \
 		protoc -I $$dir --go_out=plugins=grpc:$$dir $$dir/*.proto; \
 	done
+	cat grpclb/grpc_lb_v1/load_balancer.pb.go | \
```

Comment:
Why would we do that, given that this work is already done, and is correct?

Feel free to add the `go:generate` lines, but that doesn't seem to be within the scope of this PR, to me.